File: src/main/java/secret/SecretPart.java
package secret;

import java.math.BigInteger;

public class SecretPart {
    private BigInteger[][] matrix;
    private BigInteger r;
    private String id;

    public SecretPart(BigInteger[][] matrix, BigInteger r, String id) {
        this.matrix = matrix;
        this.r = r;
        this.id = id;
    }

    public String getId() {
        return id;
    }

    public BigInteger[][] getMatrix() {
        return matrix;
    }

    public BigInteger getR() {
        return r;
    }
}

File: python/secret/SecretPart.py

class SecretPart:

    def __init__(self, matrix, r: int, id: str):
        self.__matrix = matrix
        self.__r = r
        self.__id = id

    def getMatrix(self):
        return self.__matrix

    def getR(self):
        return self.__r

    def getId(self):
        return self.__id



